sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_component_builder.py

```diff
@@ -277,7 +277,14 @@ def _map_type(_meta):
         return output_dict
 
     def _get_group_parameter_defaults(self):
-        return {key: copy.deepcopy(val.default) for key, val in self.inputs.items() if isinstance(val, GroupInput)}
+        group_defaults = {}
+        for key, val in self.inputs.items():
+            if not isinstance(val, GroupInput):
+                continue
+            # Copy and insert top-level parameter name into group names for all items
+            group_defaults[key] = copy.deepcopy(val.default)
```

Comment:
why copy GroupInput only?

Reply:
Offline synced, will refine it somehow in the next pr